docs/source/technical_tutorials/working_with_r/cmr_search_in_r.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>We could make all of this work unnecessary by adding the option to return a csv from the API (instead of json). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>